File: benchmark/php/prime_count.php
<?php
function is_prime($n) {
    if ($n < 2) return false;
    if ($n == 2) return true;
    if ($n % 2 == 0) return false;
    for ($i = 3; $i * $i <= $n; $i += 2) {
        if ($n % $i == 0) return false;
    }
    return true;
}
$start = hrtime(true);
$count = 0;
for ($n = 2; $n <= 100000; $n++) {
    if (is_prime($n)) $count++;
}
$end = hrtime(true);
$ms = ($end - $start) / 1e6;
echo "Primes: " . $count . "
";
echo "Time: " . round($ms) . "ms
";

File: benchmark/src/python/prime_count.py
import time
def is_prime(n):
    if n < 2: return False
    if n == 2: return True
    if n % 2 == 0: return False
    i = 3
    while i * i <= n:
        if n % i == 0: return False
        i += 2
    return True
start = time.time() * 1000
count = 0
for n in range(2, 100001):
    if is_prime(n):
        count += 1
end = time.time() * 1000
print(f"Result: {count}")
print(f"Time: {end - start:.0f}ms")
